calendar/quickstart/index.js

```diff
@@ -18,17 +18,19 @@
 const fs = require('fs');
 const mkdirp = require('mkdirp');
 const readline = require('readline');
-const google = require('googleapis');
+const {google} = require('googleapis');
 const OAuth2Client = google.auth.OAuth2;
 const SCOPES = ['https://www.googleapis.com/auth/calendar.readonly'];
 const TOKEN_PATH = 'credentials.json';
+const token = {};
```

Comment:
This variable doesn't need to be file-wide, put it next to the other code (l45):

`const token = {};`